docs/guide/areas-and-widgets.md

```diff
@@ -74,6 +82,88 @@ introduction: {
 }
 ```
 
+## Expanded widget preview menu configuration
+ To enhance the editor experience, an expanded widget menu can be added instead of the basic menu. This context menu expands from the left side and provides a visual indicator for each widget in the area and support for organizing widgets into groups. These visual indicators can be preview images or icons.
+
+ Adding an `area` using the expanded menu requires a [field schema](/guide/content-schema.md) slightly different from a basic area. The following example shows a landing page type with one area field named `main`.
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  extend: '@apostrophecms/page-type',
+  options: {
+    label: 'Landing Page'
+  },
+  fields: {
+    add: {
+      main: {
+        type: 'area',
+        options: {
+      👉🏻  expanded: true,
```

Comment:
Nice use of emoji

Reply:
Make sure however that it lines up reasonably in "npm run dev" (i.e. in our actual vuepress build) which might not match the behavior of github markdown when it comes to fixed with fonts and emoji

Reply:
Checked and it looks good to me.